app/models/protocol.rb

```diff
@@ -100,6 +100,12 @@ class Protocol < ActiveRecord::Base
   attr_accessible :type
   attr_accessible :udak_project_number
   attr_accessible :vertebrate_animals_info_attributes
+  attr_accessible :research_master_id
+  attr_accessible :has_human_subject_info
+
+  validates :research_master_id, numericality: true, allow_blank: true
```

Comment:
Would this allow non-integers to be entered? Shouldn't we use numericality: { only_integer: true }